### PR TITLE
Change logic for CPP search option feature flag

### PR DESCRIPTION
**Pull request by brangi**
## Description
Issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/10698

## Testing done
Locally

## Acceptance criteria
- [x] "Community care providers" is present in the  VA facility type drop down list

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/facility-locator/tests/e2e/ccpDisabled.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    cy.injectAxe();
+    cy.axeCheck();
+
+    cy.get('#facility-type-dropdown option').then(options => {
+      const optionsWithoutCCP = [...options].map(o => o.text);
+      expect(
+        optionsWithoutCCP.includes(
+          facilityTypesOptions[LocationType.CC_PROVIDER],
+        ),
+      ).to.eq(false);
```
Comment (mpelzsherman) [inline src/applications/facility-locator/tests/e2e/ccpDisabled.cypress.spec.js:34]: Can we do

```js
expect(optionsWithoutCCP).to.not.include(facilityTypesOptions[LocationType.CC_PROVIDER])
```
?

File: src/applications/facility-locator/tests/e2e/ccpDisabled.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    });
```
Comment (mpelzsherman) [inline src/applications/facility-locator/tests/e2e/ccpDisabled.cypress.spec.js:35]: add whitespace?

File: src/applications/facility-locator/tests/e2e/ccpDisabled.cypress.spec.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    cy.get('#facility-type-dropdown option').then(options => {
+      const optionsWithoutCCP = [...options].map(o => o.value);
+      expect(optionsWithoutCCP.includes(LocationType.CC_PROVIDER)).to.eq(false);
+    });
```
Comment (mpelzsherman) [inline src/applications/facility-locator/tests/e2e/ccpDisabled.cypress.spec.js:39]: I don't think this 2nd test is needed.

File: src/applications/facility-locator/tests/e2e/ccpEnabled.cypress.spec.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
+import path from 'path';
+import mockFeatureTogglesEnabled from '../fixtures/toggle-ccp-enabled.json';
+import { facilityTypesOptions } from '../../config';
+import { LocationType } from '../../constants';
+
+describe('Facility Search - CCP (community care providers) disabled', () => {
```
Comment (mpelzsherman) [inline src/applications/facility-locator/tests/e2e/ccpEnabled.cypress.spec.js:6]: Let's change the describe to say "enabled"

File: src/applications/facility-locator/tests/e2e/ccpEnabled.cypress.spec.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+      expect(
+        optionsWithoutCCP.includes(
+          facilityTypesOptions[LocationType.CC_PROVIDER],
+        ),
+      ).to.eq(true);
+    });
+    cy.get('#facility-type-dropdown option').then(options => {
+      const optionsWithoutCCP = [...options].map(o => o.value);
+      expect(optionsWithoutCCP.includes(LocationType.CC_PROVIDER)).to.eq(true);
+    });
```
Comment (mpelzsherman) [inline src/applications/facility-locator/tests/e2e/ccpEnabled.cypress.spec.js:41]: Same comments apply here:

1. Can we use to.not.include?
2. Delete the 2nd test - not needed IMO

File: src/applications/facility-locator/utils/selectors.js [hunk 6]
```diff
@@ -9,3 +9,6 @@ export const facilitiesPpmsSuppressPharmacies = state =>
 
 export const facilityLocatorFeUseV1 = state =>
   toggleValues(state)[FEATURE_FLAG_NAMES.facilityLocatorFeUseV1];
+
+export const facilitiesPpmsSuppressCommunityCare = state =>
+  toggleValues(state)[FEATURE_FLAG_NAMES.facilitiesPpmsSuppressCommunityCare];
```
Comment (mpelzsherman) [inline src/applications/facility-locator/utils/selectors.js:14]: Let's delete facilityLocatorShowCommunityCares

File: src/platform/utilities/feature-toggles/featureFlagNames.js [hunk 7]
```diff
@@ -3,6 +3,7 @@ export default Object.freeze({
   dashboardShowCovid19Alert: 'dashboardShowCovid19Alert',
   facilityLocatorShowCommunityCares: 'facilityLocatorShowCommunityCares',
```
Comment (mpelzsherman) [inline src/platform/utilities/feature-toggles/featureFlagNames.js:4]: I don't like leaving both flags in the code, as this creates confusion. I think we should at least add a comment to explain that `facilityLocatorShowCommunityCares` is deprecated, and reach out to the teams that are currently using it.

Comment (brangi) [reply src/platform/utilities/feature-toggles/featureFlagNames.js]: Yeah I don't think we should remove the flag now but adding a comment.

**Review by mpelzsherman — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Pull request merged**